src/test/java/games/strategy/ui/SwingActionTest.java

```diff
@@ -0,0 +1,51 @@
+package games.strategy.ui;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.lang.reflect.InvocationTargetException;
+import java.util.function.Consumer;
+
+import javax.swing.Action;
+import javax.swing.SwingUtilities;
+
+import org.junit.experimental.extensions.MockitoExtension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+
+@ExtendWith(MockitoExtension.class)
+public class SwingActionTest {
+
+  @Test
+  public void testActionOf(@Mock final ActionEvent event, @Mock final ActionListener listener) {
```

Comment:
Sweet syntax.  Didn't even notice the `MockitoExtension` class offered that when it was introduced.  Do we run any risk of this syntax not being supported when an "official" JUnit 5 Mockito extension is made available?

Reply:
It could happen that it's not supported this way, although chances are good, but in any case we're able to keep the current mockito extension class